gpflow/conditionals/__init__.py

```diff
@@ -3,10 +3,10 @@
 from .dispatch import conditional, sample_conditional
 
 from . import conditionals
-from . import mo_conditionals
-
 from . import sample_conditionals
-from . import mo_sample_conditionals
+
+from .multioutput import conditionals as mo_conditionals
+from .multioutput import sample_conditionals as mo_sample_conditionals
```

Comment:
why the extra aliasing ?

Reply:
I would just do `from . import multioutput` to ensure they are all registered on the Dispatcher - users shouldn't have to access the module itself otherwise